presto-hive/pom.xml

```diff
@@ -412,6 +412,17 @@
                     </ignoredDependencies>
                 </configuration>
             </plugin>
+
+            <plugin>
+                <groupId>org.gaul</groupId>
+                <artifactId>modernizer-maven-plugin</artifactId>
+                <configuration>
+                    <!-- To allow usage of com.google.common.io.BaseEncoding -->
+                    <ignorePackages>
+                        <ignorePackage>com.facebook.presto.hive.authentication</ignorePackage>
```

Comment:
Modernizer gives a legit suggesting that the https://docs.oracle.com/javase/8/docs/api/java/util/Base64.Encoder.html must be used instead of Guava.

Reply:
Yes. that is the reason I put this line here in order to use Guava B64 Encoding. So what is the actual preference, use guava or use java.util.* ?

Reply:
When I suggested using Guava i didn't realize the Base64 encoder is included into Java standard library. Java standard library is preferred according to modernizr. In most cases we are trying to follow modernizr suggestions.